dot/network/service.go

```diff
@@ -430,6 +430,25 @@ func (s *Service) sentBlockIntervalTelemetry() {
 func (s *Service) handleConn(conn libp2pnetwork.Conn) {
 	// TODO: currently we only have one set so setID is 0, change this once we have more set in peerSet.
 	s.host.cm.peerSetHandler.Incoming(0, conn.RemotePeer())
+
+	// exchange BlockAnnounceHandshake with peer so we can start to
+	// sync if necessary.
+	prtl, has := s.notificationsProtocols[BlockAnnounceMsgType]
+	if !has {
+		return
+	}
+
+	hs, err := prtl.getHandshake()
+	if err != nil {
+		return
```

Comment:
Maybe log it out? :thinking: Does this happen often?

Same question for `sendHandshake` below

Reply:
i don't think this would ever really happen, I can add a log though

Reply:
Yeah log it out at the error level so we can see if it happens? If it does happen a lot, we can just then either handle it or log it at the trace level.